docs/topics_5/web-adding-global-custom-rules.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_rules_{context}']
+= Adding global custom rules
+
+Global rules apply to all projects.
+
+The {ProductShortName} includes a preconfigured set of global rules. You can define your own custom global rules.
```

Comment:
Move the first sentence to the paragraph above as the first sentence. The meaning is more logical.

Second sentence should stand on its own because it is the subject of this section